scripts/fixtures/clinics_collection_sample.json

```diff
@@ -0,0 +1 @@
+[{"approved":true,"note":"Waiting list is currently full and limited to residents of Toshima. Voucher required. (2021-06-22)","website":"https://www.otsukaseikei.jp","name":"Otsuka Orthopedic Surgeon","city":"Otsuka","ward":"Toshima","prefecture":"Tokyo","id":"0cuHYoQzgl7fEcVVzyIN"},{"note":"Requires a voucher. (2021-06-18)","ward":"Sakai-ku","prefecture":"Osaka","name":"Nagayama Seikei Naika","approved":true,"website":"https://www.nagayama-seikei-naika.com/covid19","id":"0eFqHQaL87rA9kqLxsdt","city":"Sakai"},{"city":"Nagakute ","name":"Mass Vaccination Site","approved":true,"prefecture":"Aichi","ward":"Nagakute","website":"https://www.city.nagakute.lg.jp/soshiki/fukushibu/kenkosuishinka/4/coronayoyaku.html","id":"0sh1jTRYeVQngDJLeFZU"},{"city":"Hana-Koganei","note":"Cancellation list now requires a voucher and 65 years or older. (2021-06-21)","name":"Sasaki Clinic","prefecture":"Tokyo","ward":"Kodaira","website":"http://www.sasa-cli.com/information/detail/3538.html","id":"2ontw1VV9bnU3MqFHvte","approved":true},{"website":"https://caps-clinic.jp/nishikasai/course/covid19-vaccine/","prefecture":"Tokyo","city":"Tokyo","ward":"Edogawa","approved":true,"name":"CAPS Clinic Nishi Kasai","id":"3XH5DBIZ3rAHVoHxMYPI"},{"approved":true,"name":"Caps Clinic Shiki","website":"https://caps-clinic.jp/shiki/course/covid19-vaccine/","prefecture":"Saitama","city":"Shiki","ward":"Shiki","id":"4SvNlbsSoAV70yw7437v"},{"ward":"Onomichi ","approved":true,"website":"https://www.city.onomichi.hiroshima.jp/soshiki/19/40383.html","prefecture":"Hiroshima","name":"Onomichi City Vaccination Venue","city":"Onomichi ","id":"71xhIOlyMnLcP1bxCuaS"},{"id":"BPUEurZJYRyYn7bRojAw","city":"Tsukuba","prefecture":"Ibaraki","ward":"Matsunoki","note":"Cancellation waiting list open to anyone with or without vouchers.","name":"Sakane M Clinic","approved":false,"website":"http://www.sakanemclinic.com/"},{"id":"Dseg4TO4927EuoS2vo9X","prefecture":"Tokyo","website":"https://mobile.twitter.com/clinic_ten/status/1406789750069944320","city":"Tokyo","name":"CLINIC TEN SHIBUYA","ward":"Shibuya","note":"Vouchers are not required.\nRecommend following the clinic's account for more imformation.","approved":false},{"approved":false,"note":"Please refer to news article titled「新型コロナワクチンキャンセル待ちのお知らせ」dated 06.09 for the procedure.","ward":"Ajina","website":"https://hiratamedical.com/","id":"H67NOgj9aRLOnZ1GPoRH","name":"Hirata Medical Clinic","city":"Hatsukaichi","prefecture":"Hiroshima"}]
```

Comment:
these should move out of `scripts` since we can also use these fixtures for tests.

Reply:
Yeah, I agree for the future - hence the name =). I was thinking we'd do that when we add tests, since they're only used for scripting right now. If you'd strongly prefer I can move them in this PR though. LMK.

Reply:
I think it would be good to do in this request :)

Reply:
Done.